src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/Homepage.cs

```diff
@@ -50,5 +50,17 @@ public string LoginLogoutLinkText()
             wait.Until(s => s.FindElement(pages.Homepage.LoginLogoutLink).Displayed);
             return driver.FindElement(pages.Homepage.LoginLogoutLink).Text;
         }
+
+        public void LogOut()
+        {
+            if (LoginLogoutLinkText().Equals("Log out"))
```

Comment:
Should this be case insensitive?

Reply:
You are quite correct, fixed in next commit